### ISSUE TITLE
History paging

### ISSUE THREAD

**Issue by frogale** [hajekmi/myjablotron#2]
It would be great to have control of history paging... For now only 40 lines is returned.. :-(

**Comment by hajekmi** [comment]
Yes, I accepted this bug. Function getHistory is for me minor feature, because in my project I don't use it.
